src/components/lesson-metadata-modal.js

```diff
@@ -156,6 +206,22 @@ function LessonMetadataModal({ lesson, mode, isVisible, onSave, onClose }) {
             showTime={{ defaultValue: moment(`${firstEnabledHour}:00`, 'HH:mm'), format: 'HH:mm', hideDisabledOptions: true }}
             />
         </FormItem>
+        <FormItem name="enableSequence" valuePropName="checked" hidden>
+          <Checkbox />
+        </FormItem>
+        {mode === LESSON_MODAL_MODE.create && (
+          <Collapse className="LessonMetadataModal-sequenceCollapse" activeKey={isSequenceExpanded ? ['sequence'] : []} onChange={handleRepetitionsCollapseChange} ghost>
```

Comment:
Probably an old name here, `handleRepetitionsCollapseChange` -> `handleSequenceCollapseChange`

Reply:
Yep, renamed to `handleSequenceCollapseChange`